pyproject.toml

```diff
@@ -1,6 +1,30 @@
+[project]
+name = "vllm-spyre"
+description = "vLLM plugin for Spyre hardware support"
+readme = "README.md"
+license = {text = "Apache 2"}
+version = "0.0.0" # TODO dynamic versioning
+dependencies = [
+    "fms-model-optimizer>=0.2.0",
+    "ibm-fms==0.0.8",
+    "vllm",
+]
+
 [tool.setuptools_scm]
 # version_file = "vllm_spyre/_version.py" # currently handled by `setup.py:get_version()`
 
+[tool.uv]
+# Never install torch, so that no dependencies can override it.
+# This requires that torch is installed separately in the target environment
+override-dependencies = [
+    "torch; sys_platform == 'never'",
+    "torchaudio; sys_platform == 'never'",
+    "torchvision; sys_platform == 'never'",
+]
+
+[tool.uv.sources]
+vllm = { git = "https://github.com/vllm-project/vllm", rev = "v0.8.0" }
```

Comment:
Specifying the source for vllm allows `uv` to run the install from source, specifying the `empty` target so that the gpu deps don't get installed